operations/observability/mixins/meta/rules/server.yaml

```diff
@@ -97,6 +97,20 @@ spec:
         description: Server is consumming too much CPU. Please investigate.
         dashboard_url: https://grafana.gitpod.io/d/6581e46e4e5c7ba40a07646395ef7b23/kubernetes-compute-resources-pod?var-cluster={{ $labels.cluster }}&var-namespace=default
 
+    - alert: DashboardStuckInPodInitState
+      # Reasoning: alert if dashboard is stuck in init more than 5 minute.
+      expr: sum(increase(kube_pod_container_status_waiting_reason{container="dashboard", reason="PodInitializing"}[5m])) by (container) > 0
+      # Five minutes sound high, but that's the only value that's higher than recent history
```

Comment:
@mustard-mh What does it mean high? Do we get alert too late or we miss?

Reply:
It's a copied text from alert below xd